aws/ecs_cluster_test.go

```diff
@@ -0,0 +1,104 @@
+package aws
+
+import (
+	"testing"
+	"time"
+
+	awsgo "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/gruntwork-io/cloud-nuke/util"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const region = "eu-west-1"
```

Comment:
Nit: The convention is to use all caps to name constants to make it more obvious when reading the code that it's constant. E.g. `REGION` instead of `region`.

Reply:
In general, that's spot on, but in Go specifically, it's a bit different. Capitalized variables/functions in Go are public, and that may not be what you want. So generally, in Go, you use [mixed caps (AKA CamelCase)](https://golang.org/doc/effective_go.html#mixed-caps), even for constants. So `region` or `testRegion` for private constants and `Region` or `TestRegion` for public ones.

Note that we got this convention wrong in the past (from all prev languages, I was using all caps just as you suggested Ana!), but we've been incrementally switching to the official Go convention.

Reply:
Ah, good point about the capitalization making things public. I stand correct here - thanks for pointing this out!

Reply:
awesome - thanks for pointing this out!